library/src/test/java/com/chuckerteam/chucker/internal/support/FormatUtilsTest.kt

```diff
@@ -58,4 +88,79 @@ class FormatUtilsTest {
             """.trimIndent()
         )
     }
+
+    @Test
+    fun testFormatHeaders_withNullValues() {
+        val result = FormatUtils.formatHeaders(null, false)
+        assertThat(result).isEmpty()
+    }
+
+    @Test
+    fun testFormatHeaders_withEmptyValues() {
+        val result = FormatUtils.formatHeaders(listOf(), false)
+        assertThat(result).isEmpty()
+    }
+
+    @Test
+    fun testFormatHeaders_withoutMarkup() {
+        val result = FormatUtils.formatHeaders(exampleHeadersList, false)
+        val expected = "Accept: text/html\nAuthorization: exampleToken\n"
+        assertThat(result).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatHeaders_withMarkup() {
+        val result = FormatUtils.formatHeaders(exampleHeadersList, true)
+        val expected = "<b> Accept: </b>text/html <br /><b> Authorization: </b>exampleToken <br />"
+        assertThat(result).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatByteCount_zeroBytes() {
+        val resultNonSi = FormatUtils.formatByteCount(0, false)
+        val resultSi = FormatUtils.formatByteCount(0, true)
+        val expected = "0 B"
+        assertThat(resultNonSi).isEqualTo(expected)
+        assertThat(resultSi).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatByteCount_oneKiloByte() {
+        val oneKb = 1024L
```

Comment:
Can I ask you to write a test for the `1023L` scenario as well?
That would be interesting to make sure that the SI base works correctly.